Runtime/Scripts/MobiledgeXIntegrationHelper.cs

```diff
@@ -316,6 +317,19 @@ public async Task<bool> IsRoaming()
         }
 #endif
 
+        public async Task<Dictionary<string, string>> GetDeviceInfo()
```

Comment:
Okay, extra layer, and Caller is async already.

Reply:
Im guessing this will get sent out in RegisterClient